consul/consul/spec.yml

```diff
@@ -2,7 +2,7 @@
 name: consul
 version: 0.6.4
```

Comment:
This shouldn't be possible. The `consul version` command and the `Build` column of `consul members` are set from the same location.

As for the actual version, it's almost certainly a snapshot of the master repo between 0.6.4 and 0.7.0. I went with 0.6.4 because that's the version Steve had in his attempt at a `consul-k8s` package.